packages/contractkit/src/wrappers/GoldTokenWrapper.ts

```diff
@@ -0,0 +1,15 @@
+import { GoldToken } from 'src/generated/types/GoldToken'
+import { BaseWrapper } from 'src/wrappers/BaseWrapper'
+
+export class GoldTokenWrapper extends BaseWrapper<GoldToken> {
+  allowance = this.proxyCall(this.contract.methods.allowance)
+  balanceOf = this.proxyCall(this.contract.methods.balanceOf)
```

Comment:
There's possibly a bug with `GoldToken` where this always returns `2^256 - 1`. Can you check whether or not this is the case, and if it is, have this call `web3.eth.getBalance` instead?

Reply:
is there an issue for this?

Reply:
I will use `web3.eth.getBalance` just in case

Reply:
I believe there is an issue